src/ComputeSharp.D2D1/Shaders/Interop/Factories/D2D1AffineTransformMapperFactory.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Numerics;
+using ComputeSharp.D2D1.Interop;
+using ComputeSharp.D2D1.Shaders.Interop.Factories.Abstract;
+using ComputeSharp.D2D1.Shaders.Interop.Helpers;
+
+namespace ComputeSharp.D2D1.Shaders.Interop.Factories;
+
+/// <summary>
+/// A custom <see cref="D2D1TransformMapperFactory{T, TSelf, TParameters, TTransformMapper}"/> implementation for an affine transform.
+/// </summary>
+/// <typeparam name="T">The type of D2D1 pixel shader associated to the transform mapper.</typeparam>
+internal sealed class D2D1AffineTransformMapperFactory<T> : D2D1TransformMapperFactory<T, D2D1AffineTransformMapperFactory<T>, Matrix3x2, D2D1AffineTransformMapperFactory<T>.TransformMapper>
+    where T : unmanaged, ID2D1PixelShader
+{
+    /// <summary>
+    /// A <see cref="D2D1TransformMapperParametersAccessor{T, TParameters}"/> implementation for a fixed affine transform matrix.
```

Comment:
"fixed" as in constant/immutable?

Reply:
Yup. Renamed to "constant" in 7dcc2ca22784ad32b32f67125e7f85a7f01b08a6, as `fixed` is already a keyword 🙂